File: hw_2/Codes/Agcali_AlpTugrul.js
for(var test = 0; test < 10; test++){
    console.log("test = " + test);
    if(test === 3) break;
}

for( var outer = 0 ; outer < 10; outer++ ) { 
    console.log("outer = " +outer);
    for(var j = 0; j < 10; j++){
        if(outer===2) break;
    }
}
console.log("--------------------");
label:
for( var outer = 0 ; outer < 10; outer++ ) { 
    console.log("outer = " +outer);
    for(var j = 0; j < 10; j++){
        if(outer===2) break label;
    }
}

File: hw_2/Codes/Agcali_AlpTugrul.rs
fn main() {
    for test in 1..10 {
        println!("test = {}", test);
        if test == 3{
            break;
        }
    }
    for outer in 1..10 {
        println!("outer = {}", outer);
        for j in 1..5 {
            if outer == 3{
                break;
            }
        }
    }
    println!("---------------");
    'label: for outer in 1..10 {
        println!("outer = {}", outer);
        for j in 1..5 {
            if outer == 3{
                break 'label;
            }
        }
    }
}
